### PR TITLE
[chore] update screenshots for one drive admin guide

### PR DESCRIPTION
**Pull request by Kharonus**
- Updated the screenshots changed by new UI with redirect URI step

### CODE REVIEW

File: docs/system-admin-guide/integrations/one-drive/README.md [hunk 1]
```diff
@@ -34,57 +39,78 @@ Please note these minimum version requirements for the integration to work with
...
+To edit an existing OneDrive/SharePoint file storage hover over the name of the storage you want to edit and click it.
 
 ![Select OneDrive/SharePoint file storage in OpenProject system administration](openproject_system_guide_select_onedrive_storage.png)
 
-To update the general storage information, click the **Edit** icon next to the storage provider. To replace the Azure authentication information, click on the **Sync** icon next to the OAuth application.
+To update the general storage information, click the **Edit** icon next to the storage provider. To replace the Azure
+authentication information, click on the **Sync** icon next to the OAuth application. With changing the authentication
+information the redirect URI will get generated again and thus needs to be copied again. The redirect URI can be copied
+by clicking on the **Copy-to-Clipboard** element next to the information text, or by entering the form by clicking the
+**View** icon.
```
Comment (Kharonus) [inline docs/system-admin-guide/integrations/one-drive/README.md:80]: This was changed.

File: docs/system-admin-guide/integrations/one-drive/README.md
Comment (Kharonus) [inline docs/system-admin-guide/integrations/one-drive/README.md:1]: Most of the changes here are just formatting changes due to the autoformatter.

- table alignment
- line wrap
- trailing spaces removed

The 2 real changes are highlighted with my comments.

File: docs/system-admin-guide/integrations/one-drive/README.md [hunk 2]
```diff
@@ -34,57 +39,78 @@ Please note these minimum version requirements for the integration to work with
...
+up. If no files storages have been set up yet, a banner will tell you that there are no storages yet set up.
 
 Click the green **+Storage** button and select the OneDrive/SharePoint option.
 
 ![Add a new OneDrive/SharePoint storage to OpenProject](openproject_system_guide_new_onedrive_storage.png)
 
-A screen will open, in which you will need to add the following details for your new OneDrive/SharePoint storage. Please consult your Azure administrator and the [Drive ID guide](./drive-id-guide) to obtain respective information.
+A screen will open, in which you will need to add the following details for your new OneDrive/SharePoint storage. Please
+consult your Azure administrator and the [Drive ID guide](./drive-id-guide) to obtain respective information. Be aware,
+that the last step includes copying generated information to the Azure portal.
```
Comment (Kharonus) [inline docs/system-admin-guide/integrations/one-drive/README.md:59]: This was amended.

**Review by MayaBerd — APPROVED** [review]
@Kharonus thank you very much! All changes look good to me.

**Pull request merged**